tap_hubspot/__init__.py

```diff
@@ -38,18 +38,15 @@ class StateFields:
     offset = 'offset'
     this_stream = 'this_stream'
 
-CHUNK_SIZES = {
-    "email_events": 1000 * 60 * 60 * 24,
-    "subscription_changes": 1000 * 60 * 60 * 24,
-}
-
 BASE_URL = "https://api.hubapi.com"
 
 CONTACTS_BY_COMPANY = "contacts_by_company"
 
 CONFIG = {
     "access_token": None,
     "token_expires": None,
+    "email_chunk_size": 1000 * 60 * 60 * 24,
+    "subscription_chunk_size": 1000 * 60 * 60 * 24,
```

Comment:
Assuming this gets overridden when args are parsed? Seems a bit too magic, can you make it fall back at that point? Keeping it None here? With a constant like `DEFAULT_CHUNK_SIZE`?

Reply:
My understanding is that the args will upsert into this dictionary. 

I don't think None is the right move here. But I'll pull out the numbers as `DEFAULT_CHUNK_SIZE`